.github/workflows/minimum_dependency_checker.yml

```diff
@@ -49,9 +50,11 @@ jobs:
           printf "${{ steps.min_dep_gen_core.outputs.min_reqs }}" > featuretools/tests/requirement_files/minimum_core_requirements.txt
       - name: Run min dep generator - koalas
         id: min_dep_gen_koalas
-        uses: alteryx/minimum-dependency-generator@v2
+        uses: alteryx/minimum-dependency-generator@v3.1
```

Comment:
I think we'd want to use the "`v3` refers to the latest release in `v3.x`", similar to how other actions do tagging

We'd need to move the v3 tag to point at the 3.1 release and retag the current v3 as v3.0

Reply:
No, it's fine. I control the release for that, and don't have any updates planned

Reply:
It's fine as is but I think it'd be good practice to follow this convention